src/test/jtx/impl/AMM.cpp

```diff
@@ -823,6 +824,26 @@ pay(Account const& account, AccountID const& to, STAmount const& amount)
     jv[jss::Flags] = tfUniversal;
     return jv;
 }
+
+Json::Value
+clawback(
```

Comment:
you don't need to define this func. There is already a function defined in trust.cpp i think. You can look at Clawback test cases to see where they are